specification/onlineexperimentation/data-plane/readme.md

```diff
@@ -0,0 +1,67 @@
+# Online Experimentation
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Online Experimentation
+
+## Getting Started
+
+To build the SDKs for Online Experimentation, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Online Experimentation.
+
+```yaml
+title: Online Experimentation Data Plane
+openapi-type: data-plane
+tag: package-2025-05-31-preview
+```
+
+### Tag: package-2025-05-31-preview
+
+These settings apply only when `--tag=package-2025-05-31-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2025-05-31-preview'
+input-file:
+  - Microsoft.OnlineExperimentation/preview/2025-05-31-preview/onlineexperimentation.json
+```
+
+### Suppress non-TypeSpec SDK related linting rules
+
+These set of linting rules aren't applicable to the new TypeSpec SDK code generators so suppressing them here. Eventually we will
+opt-out these rules from running in the linting tools for TypeSpec generated swagger files.
+
+```yaml
+suppressions:
+  - code: AvoidAnonymousTypes
+  - code: PatchInOperationName
+  - code: OperationIdNounVerb
+  - code: RequiredReadOnlyProperties
+  - code: SchemaNamesConvention
+  - code: SchemaDescriptionOrTitle
```

Comment:
Can you double check the suppressions?

I think `AvoidAnonymousTypes` (if it is not from template) and `PatchInOperationName` (why we had this error at all?) could be a problem.

Reply:
Thanks. This is from template readme
https://github.com/Azure/azure-rest-api-specs/blob/525c8c6cebc5837606cb14cfcb0afdb4f2f17fa9/specification/contosowidgetmanager/data-plane/readme.md?plain=1#L53-L78